electron/src/electron-main.js

```diff
@@ -229,14 +236,100 @@ function createWindow() {
   });
 }
 
-// Gets the URLs of the nodes of all the coins and adds the hosts to allowedNodes, so that it is
-// possible to connect with them.
-ipcMain.on('setNodesUrls', (event, list) => {
-  allowedNodes = new Map();
-  list.forEach(value => allowedNodes.set(url.parse(value).host, true));
+// File with the info about the custom node URLs added by the user.
+const customNodeUrlsFile = './nodes.snd'
```

Comment:
Does electron have a global storage location? Or, at least shouldn't we store this in one of the standard app config locations?

Reply:
Electron does not have a default location for saving files, but it is possible to get the path of the system folders. Here is a list of the options Electron has: https://github.com/electron/electron/blob/master/docs/api/app.md#appgetpathname

Reply:
Can we use localStorage?

Reply:
otherwise use `appData` or `userData` from that docs link

Reply:
`localStorage` can only be used inside the app window, not the main Electron code, and, as this is a security feature similar to the CSP, I would not like to involve the app code. I modified the code so that the file is now saved inside `appData`. I also solved a small bug that made some URLs to continue to be whitelisted during the session after being replaced by the user.